apps/extension/Makefile

```diff
@@ -1,18 +1,20 @@
 # Minimum Makefile for the extension package
 TVM_ROOT=$(shell cd ../..; pwd)
+CUR_ROOT=$(shell pwd)
 PKG_CFLAGS = -std=c++11 -O2 -fPIC\
 	-I${TVM_ROOT}/include\
 	-I${TVM_ROOT}/3rdparty/dmlc-core/include\
 	-I${TVM_ROOT}/3rdparty/dlpack/include\
-	-I${TVM_ROOT}/3rdparty/HalideIR/src
+	-I${TVM_ROOT}/3rdparty/HalideIR/src\
+	-I$(CUR_ROOT)/include/
```

Comment:
let us simply inline the header into tvm_ext.cc since there is not too much content in here

Reply:
ok